app/__init__.py

```diff
@@ -15,3 +15,21 @@
 db = SQLAlchemy(app)
 
 from app import server, models
+from flasgger import Swagger
+
+app.config['SWAGGER'] = {
+    "swagger_version": "2.0",
+    "specs": [
+        {
+            "version": "1.0.0",
+            "title": "Shopcarts REST API Service",
+            "description":"This is the Shopcart API",
+            "endpoint": 'v1_spec',
+            "route": '/apidocs',
+            "basePath": '/'
```

Comment:
I don't think this is supposed to reference our api route, but the versioning: 
https://github.com/nyu-devops/lab-flask-swagger/blob/2a16bfe0a595e035261eee306bcaf43d2aeb2bbd/server.py#L37

So we probably want: 
```
            "endpoint": 'v1_spec',
            "route": '/v1/spec
```